tests/test_readMcool.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env python3
+#
+# Testing for the helper function readMcool in hicrep/utils
+#
+import pytest
+from hicrep.utils import readMcool
+
+def testReadMcool():
+    fmcool = "tests/data/human_hi-c/4DNFICQK4N8B.mcool"
+
+    #Test that readMcool throws an exception if you try to read in an .mcool file at an unavailable resolution
+    with pytest.raises(KeyError):
+        readMcool(fmcool, 12345)
+
+    #Test that the read in .mcool file has the specified resolution and the correct chromosome labels
+    cool, binSize = readMcool(fmcool, 25000)
+    assert cool.binsize == 25000,\
+        f"Cooler object .mcool file {fmcool} has different resolution than requested"
+    assert set(cool.chromnames) == set(['chr' + str(ii) for ii in list(range(1,20)) + ["X", "Y"]]),\
+        f"Cool object loaded from file {fmcool} has incorrect chromosome labels"
```

Comment:
While this is true for this test data set, this is not always guaranteed as people can have whatever they want as chromosome labels so I am not sure if this test is meaningful